docs/sources/mimir/operators-guide/run-production-environment/planning-capacity.md

```diff
@@ -154,6 +154,20 @@ For more information about disk requirements, refer to [Compactor disk utilizati
 sum(prometheus_tsdb_head_series)
 ```
 
+There are two compactor specific settings related to working with large tenants that should be considered when planning/adjusting capacity.
+
+These are:
+`-compactor.split-groups`
+`-compactor.split-and-merge-shards`
+
+For more information on how they work, refer to [Compaction algorithm]({{< relref "../architecture/components/compactor/index.md#compaction-algorithm" >}}).
+
+We currently recommend 1 shard per every 8 million active series in a tenant.
```

Comment:
Latest recommendation I'm aware of was 25M series for a tenant. We were discussing 8M internally, but we haven't switched to that yet.